### DISCUSSION TITLE
Introduce global `Testing` section in the docs

### DISCUSSION THREAD

**Discussion by dissto** [filamentphp/filament#14621] [Ideas]
How would you like for all content related to tests to be under the category `Testing` (like Core Concepts, Widgets, Actions, etc), and logically grouped sub-categories.

![Screenshot 2024-10-24 185728](https://github.com/user-attachments/assets/50bc9512-1503-45f3-8199-d2a3bb78a7c2)

The immediate benefit is, that well everything related to testing is in one place, as opposed to under each cores package documentation. Which would make searching and referencing tests and examples more easy, imho.

Just an idea though 🤓

@danharrin would you like me to PR this? 🤔

**Comment by danharrin** [comment]
hey, I'm looking to do something similar for v4 so keep an eye out there. not gonna change for v3 though
